{
    "expected_exception": "AssertionError",
    "match": "ToolMessage\\ /\\ ToolCallParam\\ mismatch!\\ \\(extra\\ ChatCompletionToolMessageParam\\ `tool_call_id`s\\)\\\nThe\\ following\\ ChatCompletionToolMessageParam\\ `tool_call_id`s\\ don't\\ have\\ a\\ corresponding\\ ChatCompletionAssistantMessageParam\\ ChatCompletionMessageToolCallParam\\ `id`\\.\\\nExtra\\ ChatCompletionToolMessageParam\\ `tool_call_id`s:\\ \\['some\\ tool\\ call\\ id\\ that\\ does\\ not\\ match\\ the\\ returned\\ tol\\ call\\ id'\\]\\\nHere\\ are\\ the\\ ChatCompletionAssistantMessageParam\\ ChatCompletionMessageToolCallParam\\ `id`s\\ that\\ we\\ found:\\ \\['call_62136354'\\]\\\n\\\nToolMessage\\ /\\ ToolCallParam\\ mismatch!\\ \\(extra\\ ChatCompletionAssistantMessageParam\\ ChatCompletionMessageToolCallParam\\ `id`s\\)\\\nThe\\ following\\ ChatCompletionAssistantMessageParam\\ ChatCompletionMessageToolCallParam\\ `id`s\\ don't\\ have\\ a\\ corresponding\\ ChatCompletionToolMessageParam\\ `tool_call_id`\\.\\\nExtra\\ ChatCompletionAssistantMessageParam\\ ChatCompletionMessageToolCallParam\\ `id`s:\\ \\['call_62136354'\\]\\\nHere\\ are\\ the\\ ChatCompletionToolMessageParam\\ `tool_call_id`s\\ that\\ we\\ found:\\ \\['some\\ tool\\ call\\ id\\ that\\ does\\ not\\ match\\ the\\ returned\\ tol\\ call\\ id'\\]\\\n"
}